docs/contrib.rst

```diff
@@ -0,0 +1,12 @@
+Optional dependencies
+=====================
+
+These features intentionally stay outside of the core stereotype implementation as they may introduce dependencies.
+
+Schematics adapter
+------------------
+
+Use this only if already depending on `Schematics <https://github.com/schematics/schematics>`_.
+
+.. autoclass:: stereotype.contrib.schematics.SchematicsModelField
+   :show-inheritance:
```

Comment:
Nice find. It's because I didn't include schematics as a dependency for the remote build (it cannot import the module if schematics isn't installed). It worked locally because I had it installed in venv.